cityofaustin/atd-vz-data
# Add council districts to crash & fatality export

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/12503

I decided to add another [stored generated column](https://www.postgresql.org/docs/current/ddl-generated-columns.html) called `council_district` that calls the function `find_council_district()` to calculate the council district for each crash. Lmk what yall think!

## Testing
**URL to test:** 
https://deploy-preview-1267--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
1. Export a csv from the crashes page.
2. Do you see the council_districts column?
3. Repeat for fatalities page.
4. You can view the new generated `council_district` column in the `atd_txdot_crashes` table as well


---
#### Ship list
